chart/openfaas/values.yaml

```diff
@@ -331,6 +331,9 @@ iam:
 prometheus:
   image: prom/prometheus:v2.51.2
   create: true
+  retention:
+    time: 15d
+    storage: 0
```

Comment:
I think `storage: 0` will be confusing.

What can we do about that?

Reply:
Let's keep retention here, and if someone wants to set `storage` then perhaps they can look it up in the table of configuration options?

Reply:
The `prometheus.retention.storage` default value has been removed from the `values.yaml` file but can still be looked up in the configuration options table.